docs/ops.html

```diff
@@ -162,9 +162,8 @@ <h4><a id="basic_ops_consumer_lag" href="#basic_ops_consumer_lag">Checking consu
 
   <h4><a id="basic_ops_consumer_group" href="#basic_ops_consumer_group">Managing Consumer Groups</a></h4>
 
-  With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. Note that deletion is only available when the group metadata is stored in
-  ZooKeeper. When using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
-  the broker handles coordination of partition handling and rebalance), the group is deleted when the last committed offset for that group expires.
+  With the ConsumerGroupCommand tool, we can list, describe, or delete consumer groups. When using the <a href="http://kafka.apache.org/documentation.html#newconsumerapi">new consumer API</a> (where
+  the broker handles coordination of partition handling and rebalance), the group can be deleted manually, or automatically when the last committed offset for that group expires.
```

Comment:
nit: break line

Also, maybe it is better to mention up front that the group must be empty?

Reply:
Thanks for the feedback. The new commit should fix these.